app/segments/drill/[id]/submission/input.js

```diff
@@ -11,90 +18,358 @@ import {
   Text,
 } from "react-native-paper";
 import { SafeAreaView } from "react-native-safe-area-context";
+import {
+  getIconByKey,
+  lookUpBaselineStrokesGained,
+  lookUpExpectedPutts,
+} from "~/Utility";
 import DrillInput from "~/components/input/drillInput";
 import DrillTarget from "~/components/input/drillTarget";
-import NavigationRectange from "~/components/input/navigationRectange";
-
-import {
-  BottomSheetModal,
-  BottomSheetModalProvider,
-  BottomSheetScrollView,
-} from "@gorhom/bottom-sheet";
-import { GestureHandlerRootView } from "react-native-gesture-handler";
-import { KeyboardAwareScrollView } from "react-native-keyboard-aware-scroll-view";
-import { AttemptData } from "~/testData";
+import NavigationRectangle from "~/components/input/navigationRectangle";
+import Loading from "~/components/loading";
+import { currentAuthContext } from "~/context/Auth";
 import Description from "./modals/description";
 
-export default function Input({ inputValues, setInputValues }) {
+/***************************************
+ * Firebase Upload
```

Comment:
wait does this PR handle firebase upload, or is it just a placeholder comment for a future PR? At first glance it looks like this PR puts the output data into react state?

Reply:
Just a place holder for a future PR, I was going to attempt to do it in this one but ran out of time and wanted to get this one merged down for the demo

Reply:
Just edit the comment to have "TODO" in it, or maybe make a github issue to help keep track I guess